spikeinterface/sorters/runsorter.py

```diff
@@ -275,6 +275,33 @@ def run_sorter_docker(sorter_name, recording, docker_image, output_folder=None,
     """ + _common_param_doc
 
 
+def read_sorter_folder(output_folder, raise_error=True):
+    """
+    Read sorting from a folder.
+    The output_folder must contain a valid spikeinterface_log.json
```

Comment:
```suggestion
    Load a sorting object from a spike sorting output folder.
    The 'output_folder' must contain a valid 'spikeinterface_log.json' file
```

Can you add docstrings too?